packages/host/app/routes/card.gts

```diff
@@ -87,12 +73,30 @@ export default class RenderCard extends Route<Model | null> {
           // query param, which will trigger a refresh of the model, which will call the model hook again.
           // The model refresh happens automatically because we have operatorModeState: { refreshModel: true } in the queryParams.
           // We have that because we want to support back-forward navigation in operator mode.
-          return model;
+          return {
```

Comment:
I'm wondering if we can somehow get rid of having `resetPasswordParams` in the model. It would be so good to keep the model clean. Can we maybe read these params in the constructor of the operator mode component, using the router service and `currentRoute`? Just a thought...

Reply:
I decided to retrieve `sid` and `clientSecret` from query params in `Auth` component, so we don't need to put `resetPasswordParams` in `OperatorModeContainer` component. Please take a look [997f5b3](https://github.com/cardstack/boxel/pull/921/commits/997f5b33867386ed29b4ec0b89f46f6ac963f9fd).